src/services/utilities.ts

```diff
@@ -1837,6 +1847,47 @@ namespace ts {
         });
     }
 
+    export function copyTrailingComments(sourceNode: Node, targetNode: Node, sourceFile: SourceFile, commentKind?: CommentKind, hasTrailingNewLine?: boolean) {

```

Comment:
I've created a helper function `getAddCommentsFunction` that is shared by the copy comments functions. Does this look better?

Reply:
@sandersn will be able to give you a better sense of what's idiomatic in TS, but I would have just used an `isTrailing` bool (because I don't see any other functions that could sensible be passed).

Reply:
I like the current state. :+1: